django/test/utils.py

```diff
@@ -338,6 +338,8 @@ def first_node(document):
                 return node
 
     want, got = strip_quotes(want, got)
```

Comment:
Strip quotes strips quotes from (so: `"'foo'"` is changed to `"foo"`), this is usefull ``compare_xml`` is used in doctests (I don't know whether this is the case). Anyways I assumed that in this case whitespace should be stripped after splitting quotes, but I'm not really 100% sure whether this is the case.